ui/app/templates/docs.hbs

```diff
@@ -0,0 +1,76 @@
+<Hds::AppFrame @hasSidebar={{@showSidebar}} @hasHeader={{false}} @hasFooter={{false}} as |Frame|>
+  <Frame.Sidebar data-test-sidebar-nav>
+    <DocfyOutput @scope="docs" as |docs|>
+      {{#let docs.children as |directories|}}
+        {{#let docs.pages as |pages|}}
+          <Hds::SideNav>
+            <:header>
+              <Hds::SideNav::Header>
+                <:logo>
+                  <Hds::SideNav::Header::HomeLink @icon="vault" @route="vault" @ariaLabel="Vault dashboard" />
+                </:logo>
+                <:actions>
+                  <Hds::SideNav::Header::IconButton @icon="home" @ariaLabel="Docs index" @route="docs" />
+                </:actions>
+              </Hds::SideNav::Header>
+            </:header>
+            <:body>
+              <Hds::SideNav::List as |Nav|>
+                <Nav.BackLink @text="Back to work (Vault UI)" @route="vault" />
+
+                <Nav.Item>
+                  <DocfyLink @to={{get (find-by "title" "Index" pages) "url"}} class="hds-side-nav__list-item-link">
+                    Index
+                  </DocfyLink>
+                </Nav.Item>
+                <Nav.Item>
+                  <DocfyLink @to={{get (find-by "title" "How to doc" pages) "url"}} class="hds-side-nav__list-item-link">
+                    How to write component docs
+                  </DocfyLink>
+                </Nav.Item>
+
+                <Nav.Title>
```

Comment:
Nice-to-have: could we add a `@count` arg here to display the total number of components there are?